enterprise/frontend/src/metabase-enterprise/whitelabel/components/IllustrationWidget/IllustrationWidget.tsx

```diff
@@ -140,15 +146,15 @@ export function IllustrationWidget({
             data={data}
             value={value}
             onChange={handleChange}
-            w={isCustomIllustration && fileName ? "8.25rem" : "100%"}
+            w={isCustomIllustration ? "8.25rem" : "100%"}
             style={{
-              flexShrink: isCustomIllustration && fileName ? 0 : undefined,
+              flexShrink: isCustomIllustration ? 0 : undefined,
             }}
             error={Boolean(errorMessage)}
           />
           {isCustomIllustration && (
             <Text truncate="end" ml="auto">
-              {fileName}
+              {fileName ? fileName : t`Remove uploaded image`}
```

Comment:
It [was discussed here](https://metaboat.slack.com/archives/C063Q3F1HPF/p1710431214111719) that when the file name is gone (since we don't store the file name), this looks a little weird. So we decided to add `Remove uploaded image` when we couldn't display the file name.